src/test/java/core/basesyntax/ReaderServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.ReaderService;
+import core.basesyntax.dao.ReaderServiceImp;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static ReaderService readerService;
+    private static List<String> testInputList;
+    private static final String wrongFilePath = "<:main:>";
+    private static final String inFilePath = "src/main/resources/test-input.csv";
+    private static final String[] testInput = new String[] {"b,banana,20",

```

Comment:
we don't need 2 fields, just use List.of(firstString, second, ...) Instead of initializing arr and converting it to list later on